src/services/utilities.ts

```diff
@@ -2930,16 +2930,16 @@ function findLinkNameEnd(text: string) {
     return 0;
 }
 
-const carriageReturnLineFeed = "\r\n";
+const lineFeed = "\n";
 /**
- * The default is CRLF.
+ * The default is LF.
  *
  * @internal
  */
 export function getNewLineOrDefaultFromHost(host: FormattingHost, formatSettings?: FormatCodeSettings) {

```

Comment:
It  would be good idea to make formatSetttings required parameter so that we know where we are ignoring formatCodeSettings intentionally.

Reply:
Will do